Automattic/vip-cli
# Expose DB and expose extra services in info table

## Description

This PR brings in two changes:
* Exposes random port to connect to database
* Rewrites the way we expose extra services in info table. The previous implementation didn't work for custom services.

## Steps to Test

Setup new dev-env and check the info table (e.g. after start). It should contain database row now:
```
 NAME               test                                                                                      
 LOCATION           /home/pavel/.local/share/vip/dev-environment/test                                         
 SERVICES           devtools, nginx, php, database, memcached, vip-search, wordpress, mu-plugins, client-code 
 NGINX URLS         http://test.vipdev.lndo.site/                                                             
 ENTERPRISE SEARCH  http://127.0.0.1:49167                                                                    
 DATABASE           127.0.0.1:49166                                                                           
 STATUS             UP      
```
